frontend/app/src/components/home/ChatMessage.svelte

```diff
@@ -722,29 +730,36 @@
     $avatar-width: toRem(56);
     $avatar-width-mob: toRem(43);
 
-    @media (hover: hover) {
-        :global(.message-bubble:hover .menu-icon) {
-            opacity: 1;
+    @keyframes show-bubble-menu {
+        0% {
+            z-index: -1;
+            opacity: 0;
         }
-
-        :global(.message-bubble:hover .menu-icon .wrapper) {
-            background-color: var(--icon-msg-hv);
+        1% {
+            z-index: 1;
+            opacity: 0;
         }
-
-        :global(.message-bubble.me:hover .menu-icon .wrapper) {
-            background-color: var(--icon-inverted-hv);
+        100% {
+            z-index: 1;
+            opacity: 1;
         }
+    }
 
-        :global(.message-bubble.crypto:hover .menu-icon .wrapper) {
-            background-color: rgba(255, 255, 255, 0.3);
-        }
+    :global(.message-bubble .menu) {
+        z-index: -1;
+        opacity: 0;
+    }
 
-        :global(.me .menu-icon:hover .wrapper) {
-            background-color: var(--icon-inverted-hv);
-        }
+    :global(.menu:has(.menu-icon.open)) {
```

Comment:
I think this would be safer as this: 

`:global(.message-bubble .menu:has(.menu-icon.open))`

as I fear that, because it's global we might accidentally hit something we don't intend to hit as it is.